src/main/java/org/osiam/storage/query/FilterExpression.java

```diff
@@ -23,24 +23,26 @@
 
 package org.osiam.storage.query;
 
+import org.osiam.storage.entities.ResourceEntity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Locale;
+
 /**
  * Representation of scim search expression.
```

Comment:
But seriously, when in doubt, just remove the javadoc. I mean, does it really explain something that's need to be explained?